artifactory/utils/dependenciesutils.go

```diff
@@ -23,13 +19,17 @@ import (
 
 const (
 	// This env var should be used for downloading the extractor jars through an Artifactory remote
-	// repository, instead of downloading directly from jcenter. The remote repository should be
-	// configured to proxy jcenter.
-	// The env var should store a server ID configured by JFrog CLI.
+	// repository, instead of downloading directly from ojo. The remote repository should be
+	// configured to proxy ojo.
+
+	// Deprecated. This env var should store a server ID configured by JFrog CLI.
 	JCenterRemoteServerEnv = "JFROG_CLI_JCENTER_REMOTE_SERVER"
-	// If the JCenterRemoteServerEnv env var is used, a maven remote repository named jcenter is assumed.
+	// Deprecated. If the JCenterRemoteServerEnv env var is used, a maven remote repository named jcenter is assumed.
 	// This env var can be used to use a different remote repository name.
 	JCenterRemoteRepoEnv = "JFROG_CLI_JCENTER_REMOTE_REPO"
+
+	// This env var should store a server ID and a remote repository in form of '<ServerID>/<RemoteRepo>'
```

Comment:
Any reason why not to have two env vars like before?
We also did the same with the private plugins, and serverId with '/' is currently

Reply:
Yes. Since now both server ID and repo are mandatories.